packages/material-ui-icons/package.json

```diff
@@ -22,8 +22,9 @@
     "react-component"
   ],
   "scripts": {
-    "build": "npm run build:icons && npm run build:babel && npm run build:copy-files",
+    "build": "npm run build:icons && npm run build:babel && npm run build:babel:es2015module && npm run build:copy-files",
     "build:babel": "babel --presets es2015,react -d ./build ./src",
+    "build:babel:es2015module": "BABEL_ENV=modules babel ./src/index.js --presets=../../../scripts/material-ui-babel-preset-es2015 --out-file ./build/index.es.js",
```

Comment:
Yes, it is possible. But it could be a complication. In the root `.babelrc` file there are many plugins that seem redundant. So, if we will choose this option, it will require to list all of the babel plugins in the 'material-ui-icons' package development dependencies.

As an alternative, it is possible to clean up changes in my PR to match the 'material-ui' package scripts.